Release notes for 1.7.7 (#8867)
istio/istio.io: content/en/news/releases/1.7.x/announcing-1.7.7/index.md

---
title: Announcing Istio 1.7.7
linktitle: 1.7.7
subtitle: Patch Release
description: Istio 1.7.7 patch release.
publishdate: 2021-01-29
release: 1.7.7
aliases:
- /news/announcing-1.7.7
---

This release contains bug fixes to improve robustness. This release note describes what’s different between Istio 1.7.6 and Istio 1.7.7

{{< relnote >}}

## Changes

- **Fixed** an issue of using explicitly empty revision flag on install.
  ([Issue #26940](https://github.com/istio/istio/issues/26940))
- **Fixed** the CA’s certificate signature algorithm to be the default algorithm corresponding to the CA’s signing key type.
  ([Issue #27238](https://github.com/istio/istio/issues/27238))
- **Fixed** an issue showing unnecessary warnings when downgrading to a lower version of Istio.
  ([Issue #29183](https://github.com/istio/istio/issues/29183))

> - **Fixed** an issue that newer control plane installations are removing permissions for `rbac.istio.io` from istiod, causing older control planes relying on that CRD group to hang on restart.

Reviewer: ```suggestion
- **Fixed** an issue causing older control planes relying on the `rbac.istio.io` CRD group to hang on restart due to the fact that newer control plane installations remove those permissions from istiod, .
```

Reviewer 2: If you just take Brians change (which I like), you'll need to remove the last `,`.

Author: Thanks for catching that. Done.

---
title: Announcing Istio 1.7.7
linktitle: 1.7.7
subtitle: Patch Release
description: Istio 1.7.7 patch release.
publishdate: 2021-01-29
release: 1.7.7
aliases:
- /news/announcing-1.7.7
---

This release contains bug fixes to improve robustness. This release note describes what’s different between Istio 1.7.6 and Istio 1.7.7

{{< relnote >}}

## Changes

- **Fixed** an issue of using explicitly empty revision flag on install.
  ([Issue #26940](https://github.com/istio/istio/issues/26940))
- **Fixed** the CA’s certificate signature algorithm to be the default algorithm corresponding to the CA’s signing key type.
  ([Issue #27238](https://github.com/istio/istio/issues/27238))
- **Fixed** an issue showing unnecessary warnings when downgrading to a lower version of Istio.
  ([Issue #29183](https://github.com/istio/istio/issues/29183))
- **Fixed** an issue causing older control planes relying on the `rbac.istio.io` CRD group to hang on restart due to the fact that newer control plane installations remove those permissions from istiod.
  ([Issue #29364](https://github.com/istio/istio/issues/29364))

> - **Fixed** a memory leak from WASM `NullPlugin` `onNetworkNewConnection`.

Reviewer 2: memory leak `in` instead of `from`?

Author: Thanks, done
